certify-mock-identity.properties

```diff
@@ -13,6 +13,7 @@ mosip.certify.cache.store.individual-id=true
 mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
 mosip.certify.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
+mosip.certify.database.name=mosip_certify_mock
 
 ## ------------------------------------------- Use specific default overriding properties ------------------------------------------------------------
```

Comment:
lets correct it to say Usecase and not use

Reply:
changed